1-org/envs/shared/terraform.example.tfvars

```diff
@@ -17,6 +17,10 @@
 // Must include the domain of the organization you are deploying the foundation.
 domains_to_allow = ["example.com"]
 
+// The list of domains that email addresses added to Essential Contacts can have.
+// The domains entered here must starts with "@"
```

Comment:
This seems like something we can do within the config rather than asking users to prefix?